math/minuit2/src/MnContours.cxx

```diff
@@ -40,75 +40,97 @@ ContoursError MnContours::Contour(unsigned int px, unsigned int py, unsigned int
    unsigned int nfcn = 0;
 
    MnPrint print("MnContours");
+   print.Debug("MnContours: finding ",npoints," contours points for ",px,py," at level ",fFCN.Up()," from value ",fMinimum.Fval());
 
    std::vector<std::pair<double, double>> result;
    result.reserve(npoints);
    std::vector<MnUserParameterState> states;
    //   double edmmax = 0.5*0.05*fFCN.Up()*1.e-3;
 
    // double toler = 0.05;
-   double toler = 0.1; // use same default value as in Minos
-
-   // get first four points
-   //   std::cout<<"MnContours: get first 4 params."<<std::endl;
+   double toler = 0.1; // use same defaut value as in Minos
+
+   // get first four points running Minos separately on the two parameters
+   // and then finding the corresponding minimum in the other
+   // P1( exlow, ymin1)  where ymin1 is the parameter value (y) at the minimum of f when x is fixed to exlow
+   // P2(xmin1, eylow)  where  xmin1 is the  the parameter value (x) at the minimum of f when y is fixed to eylow
+   // P3(exup, ymin2)
+   // P4(xmin2, eyup)
    MnMinos minos(fFCN, fMinimum, fStrategy);
 
    double valx = fMinimum.UserState().Value(px);
    double valy = fMinimum.UserState().Value(py);
 
-   MinosError meX = minos.Minos(px);
-   nfcn += meX.NFcn();
-   if (!meX.IsValid()) {
+   print.Debug("Run Minos to find first 4 contour points. Current minimum is : ",valx,valy);
+
+   MinosError mex = minos.Minos(px);
+   nfcn += mex.NFcn();
+   if (!mex.IsValid()) {
       print.Error("unable to find first two points");
-      return ContoursError(px, py, result, meX, meX, nfcn);
+      return ContoursError(px, py, result, mex, mex, nfcn);
    }
-   std::pair<double, double> ex = meX();
+   std::pair<double, double> ex = mex();
+
+   print.Debug("Minos error for p0:  ",ex.first,ex.second);
 
-   MinosError meY = minos.Minos(py);
-   nfcn += meY.NFcn();
-   if (!meY.IsValid()) {
+   MinosError mey = minos.Minos(py);
```

Comment:
```suggestion
   MinosError meY = minos.Minos(py);
```
This change reverts a change I made a few days ago to fix typo warnings. A spell checker interprets `mey` as a typo of `may`, but is happy with `maY`. Can we use `maY` as the variable name again?